pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -115,8 +115,8 @@ type DefaultFlowSpec struct {
 const (
 	DefaultFluentbitImageRepository         = "fluent/fluent-bit"
 	DefaultFluentbitImageTag                = "1.8.9"
-	DefaultFluentdImageRepository           = "ghcr.io/banzaicloud/fluentd"
-	DefaultFluentdImageTag                  = "v1.14.4-alpine-1"
+	DefaultFluentdImageRepository           = "672327909798.dkr.ecr.us-east-1.amazonaws.com/sagar-fluentd"
```

Comment:
we need to check with banzai, but I see this image is built only when tagged https://github.com/banzaicloud/logging-operator/blob/b2387f8d9445c01c0c23c119785949e8c38208f7/.github/workflows/docker_fluentd.yml, so we might need 2 prs, one for fluentd changes, and another one that uses the fluentd changes

Reply:
But to push changes in ghcr.io/banzaicloud/fluentd we don’t have access right? how we are going to push this changes?

Reply:
@Hisagar please update the pr, and set the DefaultFluentdImageTag -> v1.14.4-alpine-2. If everything looks good I'll tag the fluentd on this branch.